pages/cards/[id].tsx

```diff
@@ -49,7 +49,9 @@ export async function getStaticProps({ params }) {
 
   if (!item) {
     return {
-      props: {},
+      props: {
+        error: true,
+      },
```

Comment:
This fixes the Cypress test showing the loading card forever when the card is not found